src/adapter/variables.ts

```diff
@@ -215,7 +215,6 @@ export class VariableStore {
     this._referenceToVariables.clear();
     this._objectToReference.clear();
     this._referenceToObject.clear();
-    this._cdp.Runtime.releaseObjectGroup({ objectGroup: 'console' });
```

Comment:
This `clear()` is called on `executionContextsCleared`, which happens when pages are first loaded. The problem arises if we start setting variables before `executionContextsCleared` is received, to which we send back this instruction that releases anything we had set.

We didn't use `releaseObjectGroup` in the previous adapters, and I don't believe it's strictly necessary--any page navigation/refresh will automatically discard any objects set by the console.

Reply:
I never used this before because I wasn't sure whether it's necessary and seemed fine without. Chrome devtools uses it so we might want to look into it more at some point?

Reply:
I think there might be a different issue at hand here. I added some more logging to see.